helm_deploy/laa-fala/values/fala-production.yaml

```diff
@@ -67,7 +67,7 @@ feature_flags:
 laalaa_api:
   host: "https://laa-legal-adviser-api-production.cloud-platform.service.justice.gov.uk"
 
-allowed_hosts: ".laa-fala-production.apps.live-1.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk"
+allowed_hosts: ".laa-fala-production.apps.live-1.cloud-platform.service.justice.gov.uk .find-legal-advice.justice.gov.uk .cloud-platform.service.justice.gov.uk"
```

Comment:
@psweeting1 Did we have to add .cloud-platform.service.justice.gov.uk here to get this working? Would this add any application hostted on cloud platform as an allowed_host?

Reply:
This shouldn't be in production, it was a test. It should only be in UAT and Production.